documentation/book/assembly-maintenance-time-windows.adoc

```diff
@@ -0,0 +1,22 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-maintenance-time-windows-{context}']
+
+= Maintenance time windows
+
+Using maintenance time windows you can control when Kafka and Zookeeper clusters start rolling updates.
```

Comment:
This is misleading ... as you might have many rolling updates outside of these windows.

Reply:
I saw the IMPORTANT statement. But I do not think it is good practice to say something what is not true just because you fix it later with IMPORTANT statement.

Reply:
so let's see if @laidan6000 can help us on that